google/cloud/spanner/keys.h

```diff
@@ -202,8 +246,65 @@ class KeySet {
   bool IsAll() const { return all_; }
 
  private:
+  template <typename RowType>
+  friend class KeySetBuilder;
+
+  friend ::google::spanner::v1::KeySet internal::ToProto(KeySet const& keyset);
+
   explicit KeySet(bool all) : all_(all) {}
 
+  class ValueRow {
+   public:
+    template <std::size_t SIZE>
+    explicit ValueRow(std::array<Value, SIZE> const& column_values)
+        : column_values_(
+              std::vector<Value>(column_values.begin(), column_values.end())) {}
```

Comment:
I think the explicit call to vector's constructor here is redundant and can probably be removed?

Reply:
Done.